### PR TITLE
Added rmdir function for compatable with fs.rmdir

### PR DESCRIPTION
**Pull request by meefik**
Added rmdir function as clone of unlink function. Function fs.unlink unsupported delete of directory, this fix for better compatibility with fs module.

### CODE REVIEW

File: source/index.js [hunk 1]
```diff
@@ -184,6 +184,15 @@ function createWebDAVfs(webDAVEndpoint, username, password) {
                 .catch(callback);
         },
 
+        rmdir: function(targetPath, callback) {
```
Comment (perry-mitchell) [inline source/index.js]: Please place this method in alphabetical order.

**Comment by perry-mitchell** [comment]
Thanks @meefik. This is definitely a wise addition. I think in the next major version might have to validate targets before deletion if both `unlink` and `rmdir` are to be used in different ways (files vs directories). A true fs module would differentiate between files and directories, whereas this library currently does not. Need to consider whether or not it's worthwhile making it 2 requests instead of 1.

File: source/index.js (hunk 1 above)
Comment (meefik) [inline source/index.js]: Done

**Comment by perry-mitchell** [comment]
Thanks @meefik

**Pull request merged**